src/tiled/command.cpp

```diff
@@ -30,39 +30,70 @@
 using namespace Tiled;
 using namespace Tiled::Internal;
 
+QString Command::finalWorkingDirectory() const
+{
+    QString finalWorkingDirectory = workingDirectory;
+
+    finalWorkingDirectory = replaceVariables(finalWorkingDirectory, false);
+
+    QString finalExecutable = replaceVariables(executable);
+    QFileInfo mFile(finalExecutable);   //Check if executable is a path or not
```

Comment:
Yes exactly. That's why I left the `PATH` case out. This was also mentioned in the issue ticket.